chapters/en/slides/module3/module3_05.md

```diff
@@ -6,80 +6,95 @@ title: 'module3_05'
 # PUSHing changes from a remote repository
 
 ---
-## Pushing the commit to send the changes to the remote repository on GitHub
+
+## Pushing the changes to GitHub
 
 <center>
 
 <img src='/module3/vc-push.png' width="90%" alt="404 image"/>
 
-<center/>
+</center>
 
-Notes: Once you have made one or more commits that you want to share with your collaborators, you need to push (i.e., send) those commits back to GitHub. This updates the history in the remote repository (i.e., GitHub) to match what you have in your local repository. Now when collaborators interact with the remote repository, they will be able to see the changes you made. And you can also take comfort in the fact that your work is now backed up in the cloud!
+Notes:
+Once you have made one or more commits that you want to share with your collaborators, you need to push (i.e., send/upload) those commits back to GitHub. This updates the history in the remote repository (i.e., GitHub) to match what you have in your local repository. Now when collaborators interact with the remote repository, they will be able to see the changes you made. You can also take comfort in the fact that your work is now backed up in the cloud!
 
 ---
-## 🙌 JupyterLab: Pushing the commits to GitHub  
+
+## 🙌 JupyterLab: Pushing the commits to GitHub
```

Comment:
Now that you are reading the full course probably you have an opinion about these emojis. Are other chapters having this kind of "practical" slides with demonstrations?

Reply:
Yes, I think there are places where we could add in these in other chapters, but I haven't focused on it in this review since I am already pretty slow going through everything. I think my vote would be to remove the :raised_hands:  from the slide title and then add something like ":raised_hands: Try it yourself!" only in the transcript since this makes the formatting of the slides look similar across all chapters while still allowing for the encouraging messages. 

Ideally maybe we would go through and add similar things to all chapters, but I am thinking that this is not a high priority item even if it would be nice. What do you think?